samples/sensor/mpu6050/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _mpu6050:
+
+MPU6050: Invensense Motion Tracking Device
+##########################################
+
+Description
+***********
+
+This sample application periodically (0.5 Hz) measures samples the
```

Comment:
Sigh.  Again, docs are appearing in two different PRs.  Not sure which PR is going to win.

Delete ``measures``